### PR TITLE
Fix form api create without post action parameter

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
We change default submit action for UI of form https://github.com/mautic/mautic/pull/12813
But for API, it make BC, when it require message action too.
This PR added default return post action for API request, If not set.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Download API library
3. Try create new form without postAction parameter

```
$mauticApi        = new \Mautic\MauticApi();
$api =  $mauticApi->newApi('forms', $auth, 'https://03c1-95-103-252-55.ngrok-free.app');
$formData = [
    'name' => 'Your Form Name',
    'description' => 'A brief description of your form',
    'formType' => 'standalone', // or 'campaign' based on your need
    'fields' => [
        [
            'label' => 'First Name',
            'type' => 'text', // the type of field
            'alias' => 'first_name', // the alias used to identify the field
            'required' => true, // whether the field is required
        ],
        [
            'label' => 'Email',
            'type' => 'email',
            'alias' => 'email',
            'required' => true,
        ],
        // Add more fields as needed
    ],
];

// Convert the array to JSON

// Now, you would use cURL or a library like Guzzle to send this data to the Mautic API endpoint.

$response = $api->create($formData);
```

It should work after patch

### CODE REVIEW

File: app/bundles/FormBundle/Controller/Api/FormApiController.php [hunk 1]
```diff
@@ -327,4 +327,16 @@ protected function createFieldEntityForm($entity)
             ]
         );
     }
+
+    public function newEntityAction(Request $request): Response
+    {
+        // Check for an email to see if the lead already exists
```
Comment (mollux) [inline app/bundles/FormBundle/Controller/Api/FormApiController.php]: this seems like a copypasta from `CompanyApiController::newEntityAction()`
```suggestion
```

**Review by mollux — CHANGES_REQUESTED** [review]
code looks good, some small comment inconsistency

**Comment by escopecz** [comment]
I started [the API Library tests](https://github.com/mautic/api-library/actions/runs/7970582572) on this branch

**Comment by escopecz** [comment]
The API Library tests ⬆️ are passing

**Review by escopecz — APPROVED** [review]
Thanks! 👍

**Review by mollux — APPROVED** [review]

**Pull request merged**